module-audio/tags_fetcher/TagsFetcher.cpp

```diff
@@ -1,79 +1,76 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "TagsFetcher.hpp"
-
-#include <gsl/util>
 #include <Utils.hpp>
-
 #include "fileref.h"
-#include "tag.h"
-#include "tfilestream.h"
 
 namespace tags::fetcher
 {
-    std::optional<Tags> fetchTagsInternal(std::string filePath)
+    namespace
     {
-        TagLib::FileRef const tagReader(filePath.c_str());
-        if (!tagReader.isNull() && (tagReader.tag() != nullptr)) {
-            TagLib::Tag *tags                   = tagReader.tag();
-            TagLib::AudioProperties *properties = tagReader.audioProperties();
+        std::string getTitleFromFilePath(const std::string &path)
+        {
+            const auto pos = path.rfind('/');
```

Comment:
Is this always an absolute path and thus always contains a `/`?

Reply:
It seems so, but the case when `/` is not found is also handled, whole content of `path` will be displayed then.

Reply:
Right, I forgot about it.